### PR TITLE
Create survey 'send' integration

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

**Comment by hawx** [comment]
This should have a test.

**Comment by hawx** [comment]
Will need the version changing, `2.1.0`?

File: src/main/java/esendex/sdk/java/model/domain/impl/TemplateFields.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package esendex.sdk.java.model.domain.impl;
+
+public class TemplateFields {
+
+    private String name;
+    private String value;
+
+    public TemplateFields(String name, String value)
```
Comment (hawx) [inline src/main/java/esendex/sdk/java/model/domain/impl/TemplateFields.java]: Surely `TemplateField` instead?

Comment (jackthorley) [inline src/main/java/esendex/sdk/java/model/domain/impl/TemplateFields.java]: Good point - changed.

**Pull request merged**